### PR TITLE
MacPass: updated to newest commit

### PR DESCRIPTION
**Pull request by Janosch**
#### Description

Macpass does not get new releases since two years but gets new commits on master regularly so I am updating to the latest commit which fixes some bugs.

I also fixed the maintainer line and reduced an unneccesary large indent.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.5.1
Xcode 12.4

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by Janosch** [comment]
A note for reviewers: This port only builds for macOS >= 11, therefore it was expected that the azure pipeline fails.

File: security/MacPass/Portfile [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
-                        sha256  656d090cca863b236b7d319df4a5853b55ebf9af39a31a08cb471122c9bfbf56 \
-                        size    27575
+checksums           3cfbbb1f16437b2e1ae01ecde303ca1266960f8c.tar.gz \
+                    rmd160  61549b619d1dd0b4c0441004478594971ddfd4d8 \
+                    sha256  b70559e9b20316bf74cf6ea6bd2a13bf19d8c4abfac79e3056bda0264f97973f \
+                    size    11002723 \
+                    a996a0a3980de926168df7c2f8b02a1231760561.tar.gz \
+                    rmd160  12ce0935b45c9b8a1e258f267ff8d18f0429eed0 \
+                    sha256  656d090cca863b236b7d319df4a5853b55ebf9af39a31a08cb471122c9bfbf56 \
+                    size    27575
```
Comment (reneeotten) [inline security/MacPass/Portfile:40]: since you defined the hash values already above I would probable keep that variable name here as you had it before

**Review by reneeotten — CHANGES_REQUESTED** [review]
one minor comment; however, you should separate functional changes from whitespace changes to keep the diffs readable. So please have this in two commits: (1) update and maintainer line and (2) whitespace changes

File: security/MacPass/Portfile (hunk 1 above)
Comment (Janosch) [reply security/MacPass/Portfile:40]: You are right. I missed that while doing copy & paste.

**Review by reneeotten — APPROVED** [review]

**Pull request merged**